code/modules/swarmers/swarmer_objs.dm

```diff
@@ -87,7 +87,7 @@
   * * swarmer - The swarmer being released and told what to do
   */
 /obj/structure/swarmer_beacon/proc/release_swarmer(mob/swarmer)
-	to_chat(swarmer, "<span class='bold'>SWARMER CONSTURCTION COMPLETED.  OBJECTIVES:\n\
+	to_chat(swarmer, "<span class='bold'>SWARMER CONSTRUCTION COMPLETED.  OBJECTIVES:\n\
 	                     1. CONSUME RESOURCES AND REPLICATE UNTIL THERE ARE NO MORE RESOURCES LEFT\n\
 						 2. ENSURE PROTECTION OF THE BEACON SO THIS LOCATION CAN BE INVADED AT A LATER DATE; DO NOT PERFORM ACTIONS THAT WOULD RENDER THIS LOCATION DANGEROUS OR INHOSPITABLE\n\
 						 3. BIOLOGICAL RESOURCES WILL BE HARVESTED AT A LATER DATE: DO NOT HARM THEM\n\
```

Comment:
You might want to put periods at the ends of each of these commands. Also, would it be possible to reorder them in order of descending priority, in case people confuse them with silicon laws and think they can ""outsmart"" their ""laws"" by using big brain plays to harm biological resources in order to ensure protection of the beacon?

Reply:
They are literally unable to cause harm, and the lack of periods is a choice